### PR TITLE
[Validator] Missing translations for Estonian (et)

### LINKED ISSUE #53766: Missing translations for Estonian (et)
Hello,

There are some translation work needed for Estonian (et) and we are looking for a **native** speaker to help us out. 

Here is a [short example](https://symfony-translations.nyholm.tech/#pr) of what you need to do. There are 4 rules: 

1. You must be a Estonian (et) native speaker
2. You must look at the existing translations and follow the same "style" or "tone"
3. You must make your PR to branch 5.4
4. You must use the correct indentation (number of spaces)

These are the files that should be updated: 
- [src/Symfony/Component/Validator/Resources/translations/validators.et.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Resources/translations/validators.et.xlf)

<details>
<summary>Show related strings</summary>

#### src/Symfony/Component/Validator/Resources/translations/validators.et.xlf

| Id | English | Translation | Status |
| -- | -- | -- | -- |
| 37 | This is not a valid IP address. | See väärtus ei ole kehtiv IP-aadress. | Needs review |
| 51 | No temporary folder was configured in php.ini. | php.ini-s ei olnud seadistatud ajutist kausta, või seadistatud kaust ei eksisteeri. | Needs review |
| 59 | This is not a valid International Bank Account Number (IBAN). | See väärtus ei ole kehtiv Rahvusvaheline Pangakonto Number (IBAN). | Needs review |
| 81 | This is not a valid Business Identifier Code (BIC). | See väärtus ei ole kehtiv Äriühingu Tuvastuskood (BIC). | Needs review |
| 83 | This is not a valid UUID. | See väärtus ei ole kehtiv UUID. | Needs review |

</details>

> [!NOTE]
> If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by bifidokk**
| Q             | A
| ------------- | ---
| Branch?       |  5.4 
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 
| Issues        | Fix #53766 
| License       | MIT

### CODE REVIEW

**Review by Nyholm — APPROVED** [review]
Lovely. Thank you!

File: src/Symfony/Component/Validator/Resources/translations/validators.et.xlf [hunk 1]
```diff
@@ -312,15 +312,15 @@
             </trans-unit>
             <trans-unit id="81" resname="This is not a valid Business Identifier Code (BIC).">
                 <source>This value is not a valid Business Identifier Code (BIC).</source>
-                <target state="needs-review-translation">See väärtus ei ole kehtiv Äriühingu Tuvastuskood (BIC).</target>
+                <target>See väärtus ei ole kehtiv BIC-kood (BIC).</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.et.xlf]: what about removing the double BIC?
```suggestion
                <target>See väärtus ei ole kehtiv BIC-kood.</target>
```

Comment (bifidokk) [reply src/Symfony/Component/Validator/Resources/translations/validators.et.xlf]: yes, it looks redundant. Removed

**Comment by nicolas-grekas** [comment]
Thank you @bifidokk.

**Pull request merged**